Revisions to new change version topic (#7660)
magento/devdocs: src/cloud/project/project-conf-files_services.md

To maintain Magento store security, update installed software versions before they reach EOL.

## Change service version

You can upgrade the installed service version for compatibility with the Magento version deployed in your Cloud environment.

You cannot downgrade the service version for an installed service directly. However, you can create a new service with the required version. See [Downgrade service version](#downgrade-version).

Use the [Service versions](#service-versions) table to check service version compatibility by Magento version. Note that some service versions supported by {{ site.data.var.ee }} are not supported on {{ site.data.var.ece }}.

{:.bs-callout-warning}
You must submit a support ticket to change the service configuration on Pro Production and Staging environments.

{:.bs-callout-info}
If you change the Elasticsearch service version, check the Elasticsearch composer package for compatibility with the new version. See [Elasticsearch software compatibility]({{ site.baseurl}}/cloud/project/project-conf-files_services-elastic.html#elasticsearch-software-compatibility).

### Upgrade installed service version

> You can upgrade the installed service version by changing the [`type`](#type) value for the service.

Author: ```suggestion
You can upgrade the installed service version by updating the service configuration in the `services.yaml` file.
```
